exporter/exporter_test.go

```diff
@@ -880,6 +880,9 @@ func TestImportingJobs_JobList(t *testing.T) {
 							UserName:             "user@domain.com",
 							ServicePrincipalName: "0000-1111-2222-3333-4444-5555",
 						},
+						EmailNotifications: &jobs.EmailNotifications{
+							OnFailure: []string{"user@domain.com"},
+						},
```

Comment:
what does this match against? for case insensitivity.

Reply:
it will match against the `userName`, but will do it in a case-insensitive manner.